cli/openapi3_test.go

```diff
@@ -117,32 +106,118 @@ var _ = Describe("OpenAPI v3", func() {
 			properties := transformJSON(root.Components.Schemas["lemon"].Value.Properties)
 			Expect(properties).To(Equal(expectedProperties))
 		})
-	})
 
-	Describe("Schemas", func() {
-		It("should embed schemas from properties with relation", func() {
-			expectedSchema := transformJSON(&openapi3.SchemaRef{Value: &openapi3.Schema{
-				Type: "object",
-				Properties: map[string]*openapi3.SchemaRef{
-					"id": {Value: &openapi3.Schema{
-						Description: "Shop ID description",
-						Type:        "string",
-						ExtensionProps: openapi3.ExtensionProps{Extensions: map[string]interface{}{
-							"title": "Shop ID title",
-						}},
+		It("can have additionalProperties", func() {
```

Comment:
This test case is moved and renamed pre-existing `should include additionalProperties`.